app/views/users/show.html.erb

```diff
@@ -30,7 +30,6 @@
     <% end %>
   </style>
   <div class="user-profile-header" style="<%= user_colors_style(@user) %>">
-    <a href="https://dev.to/<%= user_url(@user) %>"></a>
```

Comment:
This href was not being shown on the page. Hence I removed it. Hopefully it wasn't there for some other weird reason that I'm unaware of?